src/OpenTracing/SpanOptions.php

```diff
@@ -18,12 +18,12 @@
     private $references = [];
 
     /**
-     * @var Tag[]
+     * @var array
      */
     private $tags = [];
 
     /**
-     * @var mixed
+     * @var int|float|\DateTime
      */
     private $startTime;
 
```

Comment:
Okay, I understand that this abstract the validations. But just for the record, opentracing-javascript also validates the input and JavaScript also supports numeric keys.
I still think though that `child_of` should be camelCased, this looks like an artifact from ruby/python